pkg/reconciler/knativeeventing/knativeeventing.go

```diff
@@ -282,37 +282,31 @@ func (r *Reconciler) deleteObsoleteResources(manifest *mf.Manifest, instance *ev
 		return err
 	}
 
-	// Remove the apiserversources.sources.eventing.knative.dev CRD at 0.13
-	resource.SetAPIVersion("apiextensions.k8s.io/v1beta1")
-	resource.SetKind("CustomResourceDefinition")
-	resource.SetName("apiserversources.sources.eventing.knative.dev")
-	if err := manifest.Client.Delete(resource); err != nil {
-		return err
-	}
-
-	// Remove the containersources.sources.eventing.knative.dev CRD at 0.13
-	resource.SetAPIVersion("apiextensions.k8s.io/v1beta1")
-	resource.SetKind("CustomResourceDefinition")
-	resource.SetName("containersources.sources.eventing.knative.dev")
-	if err := manifest.Client.Delete(resource); err != nil {
-		return err
+	// Remove the obsolete CRDs at 0.13
```

Comment:
Is it safe to delete these CRDs? I know there were some historic concerns about deleting them wholesale as part of uninstall.